Makefile

```diff
@@ -20,10 +20,12 @@ integ-test:
 black:
 	black setup.py samtranslator/* tests/* integration/* bin/*.py
 	bin/json-format.py --write tests
+	bin/yaml-format.py --write tests
 
 black-check:
 	black --check setup.py samtranslator/* tests/* integration/* bin/*.py
 	bin/json-format.py --check tests
+	bin/yaml-format.py --check tests
```

Comment:
Check integration folder too?

Reply:
oh good call, there are also json files there, next PR do both yaml and json together